intg-denonavr/avr.py

```diff
@@ -535,6 +535,7 @@ async def async_update_receiver_data(self):
             # the last update and is still healthy now to ensure that
             # we don't miss any state changes while telnet is down
             # or reconnecting.
+            # TODO: is this still needed?
```

Comment:
TBH: I don't know. This was taken from the official HA Denon integration :-)
It would be time to rewrite the connection & update logic using a proper state-machine. It's getting more and more complex and error prone with only using state variables.